bin/.gitignore

```diff
@@ -0,0 +1 @@
+/org/
```

Comment:
This does not seem like it belongs here, we do not have a bin directory in the git repo. This must have just bin some extra gorp that ended up in your cldr directory.

Reply:
agreed, should not be here.  this shows up when an IDE is misconfigured.

Reply:
What are you running that's creating `/bin` ? as i said that sounds like a misconfiguration. but we can ignore it if need be. 

Seems like a different issue from dotfiles.